package.json

```diff
@@ -62,7 +62,7 @@
     "@netlify/blobs": "8.1.2",
     "@netlify/build": "30.1.1",
```

Comment:
we can't bump build here too at this point? this pulls in two versions of it now